.github/workflows/trivy.yml

```diff
@@ -0,0 +1,27 @@
+name: Trivy
```

Comment:
Let's name the file scan-container.yml and make this name Scan container. The workflow and goal is to have scans for containers and trivy is just a tool that may get swapped out later.